.github/ISSUE_TEMPLATE/wrong_or_missing_documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation Improvment
+about: Report wrong or missing documentation
+title: "DOC:"
+labels: ""
+
+---
+
+#### Location of the documentation
+
+[this should provide a description of what documentation you believe needs to be fixed/improved]
```

Comment:
I think we should also encourage a link to the documentation that they want to see improved.